ROSNET/DataModel/Connection.cs

```diff
@@ -1,72 +1,74 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Text;
-using System.IO;
-using ROSNET;
+using RosNet.Field;
 
-namespace ROSNET.DataModel
+namespace RosNet.DataModel
```

Comment:
```suggestion
namespace RosNet
```
No point in so many folders and namespaces, the Python and C++ implementations only have one, why should we need more?

Reply:
As for folders: I'd prefer not having 2000 lines long documents like in the Python implementation